src/sdk.ts

```diff
@@ -910,6 +934,21 @@ export default class SDK {
     return this.client.getExtensionSecretState(this.identifier);
   }
 
+  /**
+   * Returns a mapping of user_id to extension specific viewer states.
+   * If a viewer doesn't have state set, but was requested, that user will
+   * not be in the response object.
+   * @async
+   *
+   * @param users - an array of userIDs to request state for.
+   *
+   * @return {Promise<object>} Resolves on successful server request with an object that is a mapping
+   *  of userID to state. Rejects on failure.
+   */
+  public multiGetExtensionViewerState(users): Promise<object> {
+    return this.client.multiGetExtensionViewerState(this.identifier, users);
```

Comment:
We have server limits on the max number of users per request, can you add a comment and `throw` an error if there are too many users.

Would save an API round-trip on a client-side detectable error.